R/visr.R

```diff
@@ -316,6 +317,19 @@ visr.survfit <- function(
     ggplot2::theme(legend.key = ggplot2::element_blank()) +
     NULL
   
+  # Save applied function so that we don't have to guess later on
+  if (is.primitive(fun)) {
+    # Prevent special case of log being a .Primitive from breaking add_quantiles
+    primitive_call <- utils::capture.output(fun)
```

Comment:
will we have this as well with other primitive functions? Imagine I use log(-log) which is a common transformation in survival estimation?

Reply:
True, that currently breaks. I'll fix that

Reply:
fixed it